test/tests/spiped-basics/run.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+[ "$DEBUG" ] && set -x
+
+set -eo pipefail
+
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+image="$1"
+
+clientImage='buildpack-deps:buster-curl'
+# ensure the clientImage is ready and available
+if ! docker image inspect "$clientImage" &> /dev/null; then
+	docker pull "$clientImage" > /dev/null
+fi
+
+# Create an instance of the container-under-test
+serverImage="$("$dir/../image-name.sh" librarytest/spiped "$image")"
+"$dir/../docker-build.sh" "$dir" "$serverImage" <<EOD
+FROM $image
+COPY dir/keyfile /spiped/key
+EOD
+cid_d="$(docker run -d "$serverImage" -d -s '[0.0.0.0]:8080' -t 'example.com:80')"
+cid_e="$(docker run --link "$cid_d":spiped_d -d "$serverImage" -e -s '[0.0.0.0]:80' -t 'spiped_d:8080')"
+trap "docker rm -vf $cid_d > /dev/null" EXIT
+trap "docker rm -vf $cid_e > /dev/null" EXIT
```

Comment:
Traps on the same signal replace the old `trap`, so the later have to include everything any previous `trap` does.

Reply:
Thanks, even without newer traps overriding older ones not doing this in a single call is dumb. I've fixed that.